src/AdoNetCore.AseClient/AseDataReader.cs

```diff
@@ -461,7 +461,7 @@ public override string GetString(int i)
                 return new string(c, 0, c.Length - 1);
             }
 
-            if (!(obj is IConvertible convertible))
+            if (obj is DBNull || !(obj is IConvertible convertible))
```

Comment:
Note: we'll be going over this behaviour again in a fix for issue #36 (discrepancy in `AseDataReader.Get*` methods)